### ISSUE TITLE
Create process that automatically removing empty files

### ISSUE THREAD

**Issue by TheRealCasadaro** [CASADARO-LABS/CASADARO-LABS#3]
when working empty notes are created for various reasons. These empty notes don't affect the system and can be left alone or deleted when they are encountered.  Because these notes contain no valuable information and are often untiled there is no reason they should not be automatically removed. This way one knows that every note they see contains something that is useful.

**Comment by TheRealCasadaro** [comment]
To keep from exposing private notes to the public, add the DAILY NOTES to the repository `.gitignore` file. 
This should be done in such a way that the folder can be seen in the repository but, the only file in the folder that can be seen is a README.md file that describes the role that the containing folder plays in the system.

**Comment by TheRealCasadaro** [comment]
I decided to hold off on creating something to auto delete empty files because I realized that some empty files have a lot of other files pointing to them; I realized that this could act as sort of a ranking system for things that I might want to study more, write about, or record a broadcast on.  These files also seem to act as hubs to other information points. To me, these hubs are important because they are unique from person to person.
